tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/JoinClauseTest.java

```diff
@@ -26,22 +26,41 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static org.ballerinalang.test.util.BAssertUtil.validateError;
+
 /**
  * This contains methods to test join clause in query expression.
  *
  * @since 1.3.0
  */
 public class JoinClauseTest {
     private CompileResult result;
+    private CompileResult negativeResult;
 
     @BeforeClass
     public void setup() {
         result = BCompileUtil.compile("test-src/query/join-clause.bal");
+        negativeResult = BCompileUtil.compile("test-src/query/join-clause-negative.bal");
+    }
+
+    @Test(description = "Test join clause with record variable definition type 1")
```

Comment:
Normally description should say what you are testing/testing preconditions/data etc. Don't repeat the same thing as in the test name

Reply:
Fixed in 74bd1c3